core/common/src/rpc/IModelTileRpcInterface.ts

```diff
@@ -60,9 +60,19 @@ export abstract class IModelTileRpcInterface extends RpcInterface {
     return this.forward(arguments);
   }
 
+  /** Returns true if an external tile cache is configured on the backend.
+   * @internal
+   */
+  public async isUsingExternalTileCache(): Promise<boolean> {

```

Comment:
@swbsi if I finally understand correctly, I should be able to use `getTileCacheContainerUrl` and determine that it's "empty", instead of adding a new method - correct?

Reply:
Yes (assuming you don't mind making the backend call to find out).

Reply:
Either way I have to make a backend call - only the backend knows if a cache has been configured. What should I be looking for - an empty url string in the returned CloudStorageContainerUrl?

Reply:
Yes that's the right check.

Reply:
Why does `getTileCacheContainerUrl` take an IModelRpcProps? It doesn't use it.

Reply:
Forget it I'll keep `isUsingExternalTileCache`. I don't have an `id` to supply. The complete lack of documentation is a problem.